superset/datasets/commands/duplicate.py

```diff
@@ -122,7 +120,7 @@ def validate(self) -> None:
             exceptions.append(DatasetExistsValidationError(table_name=duplicate_name))
 
         try:
-            owners = self.populate_owners(self._actor)
```

Comment:
See the previous comment. I gather Mypy didn't barf because it was being passed `List[int]` as opposed to `User` per the class `__init__` method. 

Note, per [here](https://github.com/apache/superset/blob/f0ca158989644b793719884b52d04f93c05de1ba/superset/commands/base.py#L58) there's no need to specify `[g.user.id]` given that it defaults to the current user if not specified.